app/models/manager_refresh/inventory_collection.rb

```diff
@@ -1,3 +1,50 @@
+# Usage:
+####################################################################################################################
+# Example 1, storing Vm model data into the DB:
+####################################################################################################################
+#   ################################################################################################################
+#   # Example 1.1 Starting with no vms, lets add vm1 and vm2
```

Comment:
sure - I havent touched the docs at all, just moved it, to produce at least _some_ output.
First wanted to make sure we want to go in that direction...